zzzz_modular_occulus/code/modules/projectiles/guns/energy/dssguns.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/weapon/gun/energy/resolute
+	name = "DSS ECAR \"Resolute\""
+	desc = "\"DeepFlare Security System\" energy carbine. The byzantine internal capacitor design requires this gun to fire in bursts of three. It has notably become a go-to firearm for station security forces due to it's stun capabilities."
+	icon = 'zzzz_modular_occulus/icons/obj/guns/energy/resolute.dmi'
+	item_state = "vulprifle"
+	icon_state = "vulprifle"
+	item_charge_meter = TRUE
+	suitable_cell = /obj/item/weapon/cell/medium
+	w_class = ITEM_SIZE_BULKY
+	force = WEAPON_FORCE_PAINFUL
+	twohanded = TRUE
+	one_hand_penalty = 10
+	slot_flags = SLOT_BELT
+	recoil_buildup = 5
+	charge_cost = 40
+	damage_multiplier = .9
+	price_tag = 2500
+	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_PLASTIC = 8, MATERIAL_SILVER = 7, MATERIAL_GOLD = 4)
+	fire_sound = 'sound/weapons/Laser.ogg'
+	rarity_value = 8
+	init_firemodes = list(
+		list(mode_name="stun", burst=3, projectile_type=/obj/item/projectile/beam/stun, fire_sound='sound/weapons/Taser.ogg', fire_delay=8, charge_cost=40, icon="stun", projectile_color = "#00FFFF"),
+		list(mode_name="kill", burst=3, projectile_type=/obj/item/projectile/beam, fire_sound='sound/weapons/Laser.ogg', fire_delay=8, charge_cost=60, icon="kill", projectile_color = "#8bbdd9"),
+		)
+
+/obj/item/weapon/gun/energy/stalwart
+	name = "DSS ESG \"Stalwart\""
+	desc = "\"DeepFlare Security System\" tri-barrel energy scattergun. The stalwart comes with a built in bayonet and fires short tripple bursts of energized particles. "
```

Comment:
Triple with a single p.

Reply:
Aaah. Thaat would be my keyboard double typing. Heck. Thanks.